benchmarks/struct_indices/spider/generate_sql.py

```diff
@@ -5,7 +5,7 @@
 import os
 import re
 
-from langchain import OpenAI
+from langchain.llms import OpenAI, OpenAIChat
```

Comment:
so this is a bit confusing but the  langchain chatgpt wrapper that has more custom support is ChatOpenAI:
```
from langchain.chat_models import ChatOpenAI
```

integrates with their new prompt selector abstraction

Reply:
I see. I will make the selection switchable through cmd args.